### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dietertaotesting**
Updating links for informational purposes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +36,9 @@ Clone repository
 
     git clone https://github.com/oat-sa/package-tao.git
     
-Install via composer missing library and extensions
+Install via composer missing library and extensions. Composer version 2 doesn't work so you've to use version 1.
```
Comment (mike-ionut-mihai-sandu-tao) [inline README.md:39]: this is available only for sprint 121, 
for November release we will have composer 2.*

File: README.md [hunk 2]
```diff
@@ -36,8 +36,9 @@ Clone repository
...
 
+    composer self-update --1
```
Comment (mike-ionut-mihai-sandu-tao) [inline README.md:41]: maybe we don't this here as we have have it in the (https://www.taotesting.com/user-guide/installation-and-upgrade/prerequisites/)

File: README.md (hunk 1 above)
Comment (dietertaotesting) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (dietertaotesting) [reply README.md]: fixed

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
lgtm

**Comment by mike-ionut-mihai-sandu-tao** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**